src/components/Navbar.tsx

```diff
@@ -60,6 +60,9 @@ export const Navbar = () => {
         setIsMobile(window.innerWidth <= 768);
     };
 
+    const userName = currentUser?.displayName?.split(" ") ?? ["", ""];
```

Comment:
We should probably want to get the name from the user app if display name is undefined. It is from the same object returned by `useAuth`, its called `userApp`

Reply:
deez nutz

Reply:
updated